k8s/business-create-ui.cmap.param

```diff
@@ -0,0 +1,18 @@
+#=========================================================
+# Template parameters for:
+# Project: business-create-ui
+# Component: config map
+# Action: build
+# Template File: templates/business-create-ui.cmap.json
+#=========================================================
+NAME="business-create"
+TAG_NAME="dev"
+LEGAL_API_URL="https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/businesses/"
+AUTH_API_URL="https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/"
+BUSINESSES_URL="https://dev.bcregistry.ca/cooperatives/auth/"
+PAY_API_URL="https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/"
+DASHBOARD_URL="https://dev.bcregistry.ca/cooperatives/"
+KEYCLOAK_CONFIG_URL="https://dev.bcregistry.ca/cooperatives/auth/config/kc/keycloak.json"
```

Comment:
This name was changed to KEYCLOAK_CONFIG_PATH. Please search globally.

Reply:
> @kialj876 Could you also please add a pipeline to deploy to Dev?

Does github actions not do that already?